### DISCUSSION TITLE
Realm migration issue (One or more errors occurred. (The process cannot access the file 'C:\Users\maker\AppData\Roaming\osu\client.db' because it is being used by another process.))

### DISCUSSION THREAD

**Discussion by DestinEcarma** [ppy/osu#20414] [Q&A] [answered]
Please drag the attach_me.zip file here!
[attach_me.zip](https://github.com/ppy/osu/files/9632472/attach_me.zip)

**Comment by peppy** [comment, marked as answer]
You should be able to restart the game and hope it works the second time.

We're tracking this issue at https://github.com/ppy/osu/issues/19216.
